src/test/datascience/.vscode/settings.json

```diff
@@ -11,7 +11,7 @@
     "python.linting.mypyEnabled": false,
     "python.linting.banditEnabled": false,
     "python.formatting.provider": "yapf",
-    "python.pythonPath": "python",
```

Comment:
Probably dont want to submit this. Although I think the test just overwrites it each time.